.github/workflows/ci-instrumentation-with-services.yml

```diff
@@ -95,11 +90,7 @@ jobs:
         with:
           gem:  "opentelemetry-instrumentation-${{ matrix.gem }}"
           ruby: "3.1"
-      - name: "Test Ruby 3.0"
-        uses: ./.github/actions/test_gem
-        with:
-          gem:  "opentelemetry-instrumentation-${{ matrix.gem }}"
-          ruby: "3.0"
+
```

Comment:
Do we want this empty line? It looks like the CI is still running correctly, but it's different from the others.

Reply:
```suggestion
```